composer.json

```diff
@@ -8,14 +8,23 @@
   "minimum-stability": "stable",
   "autoload": {
     "psr-4": {
-      "": "src/",
+      "Google\\": "src/Google/",
```

Comment:
Please change these to `src/Google/Photos` and `src/GPBMetadata/Photos` to be more tightly scoped. (Other client libraries are also using similar package names, so we should be more precise here.)